src/chromeDebugAdapter.ts

```diff
@@ -17,6 +17,7 @@ import * as errors from './errors';
 
 import * as nls from 'vscode-nls';
 import { FinishedStartingUpEventArguments } from 'vscode-chrome-debug-core/lib/src/executionTimingsReporter';
+import { version } from 'vscode';
```

Comment:
Is this used? You don't have access to the `vscode` module from here

Reply:
Removed